docs/roadmap.md

```diff
@@ -13,22 +13,22 @@ so we have a lot of things on our plate!
 Due to the nature of the product, our roadmap is doomed to evolve
 often according to the technologies around it.
 
-Here are examples:
+Here are a few examples:
 
-* a browser will support a new web API, making it relevant for
+* a browser now supports a new web API, making it relevant for
```

Comment:
in the bullet points here, the last sentence is in present tense whereas the previous ones are in future tense.

It felt better to keep everything in present tense here but another way would be to put the last bullet point in future tense

Reply:
Good catch, thanks